### ISSUE TITLE
Loss evolution for pre-trained models?

### ISSUE THREAD

**Issue by vatsalaggarwal** [RussellSB/voice-conversion-gan#10]
Hey great work! Just wondering any chance you would have the loss evolution for your pretrained models?
